metagov/metagov/plugins/github/models.py

```diff
@@ -0,0 +1,285 @@
+import requests, json, logging
+from collections import Counter
+
+import metagov.core.plugin_decorators as Registry
+from metagov.core.models import Plugin, GovernanceProcess, ProcessStatus
+from metagov.core.errors import PluginErrorInternal
+import metagov.plugins.github.schemas as Schemas
+from metagov.plugins.github.utils import (create_issue_text, close_comment_vote_text,
+    close_react_vote_text, get_jwt)
+
+logger = logging.getLogger(__name__)
+
+
+@Registry.plugin
+class Github(Plugin):
+    name = 'github'
+    config_schema = Schemas.github_app_config_schema
+
```

Comment:
I added a new static attribute for declaring the auth type. If you implement the auth views as suggested in my previous comment, then you can use auth_type to indicate to the Driver that this is an oauth-enabled plugin. Then the Driver knows that it can enable the plugin using the [authorize endpoint](https://metagov.policykit.org/redoc/#tag/Plugin-Auth).

```suggestion
auth_type = AuthType.OAUTH
```

(if you merge in without adding oauth support, you can use `auth_type = AuthType.API_KEY`... it's not exactly an API key, but its somewhere in between..)